src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -56,11 +56,13 @@ public Drive(Gyroscope gyroscope) {
     this.gyroscope = gyroscope;
     this.differentialDriveOdometry = new DifferentialDriveOdometry(this.gyroscope.getRotation2d());
```

Comment:
We will want to do a reset encoders before this line
```suggestion
    this.resetEncoders();
    this.differentialDriveOdometry = new DifferentialDriveOdometry(this.gyroscope.getRotation2d());
```